CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 ## 🎉 Documentation and Feature Updates
 - Updated README documentation updates for easier navigation and setup of the dbt package
 - Included `shopify_[source_table_name]_identifier` variable within the Shopify source package for additional flexibility within the package when source tables are named differently.
+- **New model alert**: The package now includes customer models that are based on _email_ rather than _customer_id_ ([PR #45](https://github.com/fivetran/dbt_shopify/pull/45)):
+  - `shopify__customer_emails`
+  - `shopify__customer_email_cohorts`
+  - Intermediate models that roll customer_ids up to emails
```

Comment:
It's probably worth naming the intermediate models!

Also it's unclear as to why the seed files are added--all the data looks fine but might want to mention why these seed files are now being added?

Reply:
seed data is added because of the new models i added in the [source package](https://github.com/fivetran/dbt_shopify_source/pull/39) -- in the later documentation PR, i'll delineate all the new source models that were added and note that they have seed data

Reply:
and yah i agree on the intermediate models -- maybe an expandable/collapsable dropdown for those ones would be good 🤔